custom/stdweb/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "getrandom-stdweb"
+version = "0.1.0"
+edition = "2018"
+authors = ["The Rand Project Developers"]
+license = "MIT OR Apache-2.0"
+description = "Custom shim for using getrandom with stdweb"
+documentation = "https://docs.rs/getrandom-stdweb"
+repository = "https://github.com/rust-random/getrandom/tree/master/custom/stdweb"
+categories = ["wasm"]
+
+[dependencies]
+getrandom = { path = "../..", features = ["custom"] }
```

Comment:
I think it's wroth to add `version = "0.2"`, same with the `getrandom-wasm-bindgen` crate.

Reply:
Done, also bumped `getrandom` to 0.2

Reply:
You have misunderstood me, I suggested to replace `getrandom = { path = "../..", features = ["custom"] }` with `getrandom = { version = "0.2", path = "../..", features = ["custom"] }`. I am fine with `getrandom-*` crates not synchronizing versions with `getrandom`.

Reply:
Fixed, I should have [read the docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies).